lib/bigquery/dataset.js

```diff
@@ -56,6 +56,9 @@ function Dataset(bigQuery, id) {
 /**
  * Create a table given a tableId or configuration object.
  *
+ * See [Tables: insert](https://cloud.google.com/bigquery/docs/reference/v2/tables/insert)
```

Comment:
That's what I'm thinking, it looks like JSCS was already doing line length checks, so it seems silly to do it twice.